samples/scala-eventsourced-counter/src/test/scala/com/example/actions/CounterJournalToTopicActionSpec.scala

```diff
@@ -15,6 +18,13 @@ class CounterJournalToTopicActionSpec extends AnyWordSpec with Matchers {
       result.reply shouldBe Increased(1)
     }
 
+    "handle command Increase with Metadata" in {
+      val context = new TestKitActionContext(metadataEntries = Map("myKey" -> Left("myValue")))
```

Comment:
I'm not sure I like a Left here but if not then I can't verify at compile time whether it's a String or a java.nio.ByteBuffer. 
Please, have a look at `com.akkaserveless.scalasdk.testkit.impl.TestKitActionContext`.
Another caviat is that `com.akkaserveless.javasdk.testkit.impl.TestKitActionContext` doesn't have Either so its constructor has a different signature.